examples/aws/eks-private/eks.tf

```diff
@@ -27,16 +27,20 @@ locals {
   )
 
   # Map of GPU types to their product names and instance types
-  gpu_types = {
-    "T4" = {
-      product_name   = "Tesla-T4"
-      instance_types = ["g4dn.4xlarge"]
-    }
-    "A10G" = {
-      product_name   = "NVIDIA-A10G"
-      instance_types = ["g5.4xlarge"]
-    }
-  }
+  # Additional GPU types can be added via gpu_instances.tfvars
+  gpu_types = merge(
```

Comment:
Let's not do `merge(default, additional)`.
Instead, just specify everything from the variable and this default into variable as well.